xml/obs_best_practice_image_templates.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+
+<section id="image_templates_how_to">
+ <title>Image templates</title>
+ <xi:include href="obs_image_templates.xml" xpointer="image_template_introduction_paragraph" xmlns:xi="http://www.w3.org/2001/XInclude"/>
+ <para>
+  How you can create your own image templates will be shown here.
+ </para>
+ <figure>
+  <title>OBS templates page</title>
+  <mediaobject>
+   <imageobject>
+    <imagedata fileref="image_templates_template_page_placeholder.png" />
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <section>
+  <title>Creating your own image templates</title>
+  <para>
+   Create a subproject of your home project.
+  </para>
+  <figure>
+   <title>Form for creating image template subproject</title>
```

Comment:
@mschnitzer Our documentation(s) can be found [here](http://openbuildservice.org/help/manuals/);-) And we use different styles right now for section and chapter titles right now.

And regarding the undertitles for figures, as mentioned before, they are converted to uppercase letters anyway.